lib/trans/translator.ex

```diff
@@ -74,6 +80,23 @@ defmodule Trans.Translator do
     end
   end
 
+  @spec translate!(struct, atom, String.t() | atom) :: any
+  def translate!(%{__struct__: module} = struct, field, locale)
```

Comment:
I think that it would be great to have documentation for this function. I have the goal to keep all public functions of the library documented.

The explanation added to the `translate/3` function is great but leaves `translate!/3` undocumented. I think that we could take inspiration on how Ecto documents [Ecto.Repo.one!](https://hexdocs.pm/ecto/Ecto.Repo.html#c:one!/2), which simply references [Ecto.Repo.one](https://hexdocs.pm/ecto/Ecto.Repo.html#c:one/2) and clarifies that it will raise an exception if there is more than one resulting element.

We could repeat that behaviour here having `translate!/3` reference the `translate/3` docs and simply add that it will raise an error if the translation does not exist.